src/backend/dx12/src/command.rs

```diff
@@ -360,6 +374,23 @@ impl CommandBuffer {
         }
 
         let cmd_buffer = &mut self.raw;
+
+        // Bind vertex buffers
+        // We currently don't support offsets for vertex buffer binding, therefore,
```

Comment:
why not?

Reply:
Just not implemented so far, first binding is always 0 atm.
https://github.com/gfx-rs/gfx/blob/master/src/backend/vulkan/src/command.rs#L440